prodtests/full_system_test.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# A workflow performing a full system test:
+# - simulation of digits
+# - creation of raw data
+# - reconstruction of raw data
+#
+# Note that this might require a production server to run.
+#
+# This script needs some binary objects (for the moment):
+# - matbud.root + ITSdictionary.bin + ctf_dictionary.root
+#
+# authors: D. Rohr / S. Wenzel
+
+# include jobutils, which notably brings
+# --> the taskwrapper as a simple control and monitoring tool
+#     (look inside the jobutils.sh file for documentation)
+# --> utilities to query CPU count
+. ${O2_ROOT}/share/scripts/jobutils.sh
+
+
+NEvents=${NEvents:-10} #550 for full TF (the number of PbPb events)
+NEventsQED=${NEventsQED:-1000} #35000 for full TF
+NCPUS=$(getNumberOfPhysicalCPUCores)
+echo "Found ${NCPUS} physical CPU cores"
+NJOBS=${NJOBS:-"${NCPUS}"}
+SHMSIZE=128000000000 # 128000000000  # Size of shared memory for messages
+TPCTRACKERSCRATCHMEMORY=22000000000
+NTIMEFRAMES=${NTIMEFRAMES:-1} # Number of time frames to process
+TFDELAY=100 # Delay in seconds between publishing time frames
+MCLABELS=${MCLABELS:-"--disable-mc"}
```

Comment:
We should make clear that this is not really an option but mandatory. There is no way to run with raw data and MC labels (The TPC can do this if digits are provided as inputs and on-the-fly digits-to-raw conversion is used in the TPC workflow, other detectors cannot do it at all.)

Reply:
removed